stm/stm.ml

```diff
@@ -1139,10 +1141,16 @@ let show_script ?proof () =
 
 end
 
+let doc_require_mode = function
+  | Interactive _ -> Library.RequireMostRecent
+  | Batch (Vio2Vo, _) -> Library.RequireOnlyVo
```

Comment:
Umm, aren't we supposed in `vio2vo` to be able to run in parallel thus using `vio` files?

Oh it is just the case that vio2vo will never require a file?

Reply:
vio2vo does not execute `Require`, it only executes tactics (if there is a Require in the middle of a proof, then such proof is not delayed).

Reply:
@maximedenes the thing I asked you to change is in the v2vio phase, today we load the most recent between vo and vio, while I think we should only load vio

Reply:
That's what I thought, @maximedenes , does it answer you question?

Reply:
> while I think we should only load vio

With vo as a fallback right?

Reply:
This is what happens today. But I believe it should only load vio files (at least, this is what the dependencies coqdep generates dictate)

Reply:
Internal dependencies are expected to be loaded from vio, but external ones (typically the stdlib) have only vo available right?

Reply:
I'd be OK with that, however what about system-installed libs? As of today these are mostly installed as vo-only, right?